Lab3.ipynb

```diff
@@ -0,0 +1,411 @@
+{
```

Comment:
Hi Sanae! Sorry, I don't really have any suggestions on how to make it better. Your lab report looks good to me!